plugins/workspace-minimap/src/minimap.ts

```diff
@@ -55,7 +55,11 @@ export class Minimap {
     init(): void {
       this.minimapWorkspace.scrollbar.setContainerVisible(false);
       this.primaryWorkspace.addChangeListener((e) => void this.mirror(e));
-      window.addEventListener('resize', () => {
+      Blockly.browserEvents.bind(this.minimapWorkspace.svgGroup_,
+          'mousedown', this, this.onClick);
+      Blockly.browserEvents.bind(this.minimapWorkspace.svgGroup_,
```

Comment:
One other thing is to make sure you're getting a `mousedown` event before you start responding to `mousemove` events! Otherwise someone can start a drag in one location, and then drag into the minimap, and the minimap will start updating the main workspace (which is unexpected).